src/include/misc.i

```diff
@@ -30,6 +30,22 @@ __wt_strdup(WT_SESSION_IMPL *session, const char *str, void *retp)
 }
 
 /*
+ * __wt_seconds --
```

Comment:
Then we have to have two copies, that's why I pulled out.

A bunch of the stuff in misc.i isn't really performance functions, it's more like convenience functions than anything else. We could make it a define, actually...

Anyway, if you decide it's better, I'll put two copies back into the os_xxx time source files.